packages/shared/src/utils/url/url.js

```diff
@@ -166,10 +166,10 @@ export const setUrlLanguage = lang => {
 
 export const getStaticUrl = (
     path = '',
-    is_document = false,
     options = {
         is_deriv_crypto: false,
-    }
+    },
+    is_document = false
```

Comment:
i didn't see any changes here 😅 . can you explain whats going on here

Reply:
I have switched second and third arguments.. hahaha